instagram_basic_display/InstagramBasicDisplay.py

```diff
@@ -25,7 +26,7 @@ def __init__(self, app_id: str, app_secret: str, redirect_url: str, graph_versio
         self._redirect_url = redirect_url
         self._graph_version = graph_version
 
-    def get_login_url(self, scopes: Optional[list] = None) -> str:
+    def get_login_url(self, state: Optional[str] = str(uuid.uuid4()), scopes: Optional[list] = None) -> str:
```

Comment:
Can you pls put state as a second parameter (after scopes)? Some users may use this method with positional parameters and it would be BC break for them...

Reply:
moved to keep it from being breaking change